dataservices/views.py

```diff
@@ -101,6 +119,11 @@ def get(self, *args, **kwargs):
             data={'country_data': country_data}
         )
 
+    def map_country_data(self, country):
+        return (
+            self.country_map.get(country) if self.country_map.get(country) is not None else country
```

Comment:
Again, can we use this alternative?

```suggestion
            self.country_map.get(country, country)
```